### PR TITLE
Features/home

### PR DESCRIPTION
**Pull request by molayab**
# About this PR 
> Remove all templates marks before sending a PR or it will be rejected

This PR has the UI implementation of the home scene defined in #8 (Home View Issue)

### Evidence

simulator screen shot - iphone 8 - 2018-02-11 at 01 31 37
simulator screen shot - iphone 8 - 2018-02-10 at 23 45 51
simulator screen shot - iphone 8 - 2018-02-10 at 03 47 52

### Release Notes 

- Added upcoming table
- Added custom upcoming cell actions
- Added home view scene

### CODE REVIEW

**Comment by molayab** [comment]
Please close the issue if you merge this PR, here is the number of the issue: #8 Thanks!

File: blacklist/blacklist/Scenes/Home/HomeViewController.swift [hunk 1]
```diff
@@ -93,12 +102,65 @@ extension HomeViewController: UITableViewDataSource, UITableViewDelegate {
...
+
+        let view = UIView(frame: .zero)
+
+        let headerLabel = UILabel(frame: .zero)
+        let subheaderLabel = UILabel(frame: .zero)
+
+        headerLabel.text = header
+        headerLabel.textAlignment = .center
+        headerLabel.font = UIFont(name: StyleSheet.Font.robotoRegular, size: Constants.emptyViewHeaderSize)
+        headerLabel.textColor = UIColor(hex: 0x222831)
```
Comment (santicarmo31) [inline blacklist/blacklist/Scenes/Home/HomeViewController.swift]: I think we should move this color to the constants file

File: blacklist/blacklist/Scenes/Home/HomeViewController.swift [hunk 2]
```diff
@@ -93,12 +102,65 @@ extension HomeViewController: UITableViewDataSource, UITableViewDelegate {
...
+
+        subheaderLabel.text = subheader
+        subheaderLabel.textAlignment = .center
+        subheaderLabel.font = UIFont(name: StyleSheet.Font.robotoLight, size: Constants.emptyViewSubHeaderSize)
+        subheaderLabel.textColor = UIColor(hex: 0x222831)
```
Comment (santicarmo31) [inline blacklist/blacklist/Scenes/Home/HomeViewController.swift]: Same here, the color should be on the constants file.

File: blacklist/blacklist/Scenes/Home/HomeViewController.swift (hunk 1 above)
Comment (molayab) [reply blacklist/blacklist/Scenes/Home/HomeViewController.swift]: You're right, I'll move it

File: blacklist/blacklist/Scenes/Home/HomeViewController.swift (hunk 2 above)
Comment (molayab) [reply blacklist/blacklist/Scenes/Home/HomeViewController.swift]: You're right, I'll move it

**Comment by molayab** [comment]
@santicarmo31 PR has been updated

**Pull request merged**